openpype/client/server/conversion_utils.py

```diff
@@ -1285,7 +1287,7 @@ def convert_update_representation_to_v4(
 
     if "context" in update_data:
         context = update_data["context"]
-        if "asset" in context:
+        if "folder" not in context and "asset" in context:
```

Comment:
Shouldn't we still always pop `asset` from context even if folder already exists in `context`?

Reply:
We made few mistakes, and now there are representations which have `{folder[name]}` in template bude context data have `"asset"`, then we have few representations which have `{asset}` in template but `"folder"` in data, so nope.